nixos/modules/services/hardware/pcscd.nix

```diff
@@ -16,17 +16,14 @@ let
 
 in
 {
-
-  ###### interface
-
   options.services.pcscd = {
     enable = mkEnableOption (lib.mdDoc "PCSC-Lite daemon");
 
     plugins = mkOption {
       type = types.listOf types.package;
       default = [ pkgs.ccid ];
       defaultText = literalExpression "[ pkgs.ccid ]";
-      example = literalExpression "[ pkgs.pcsc-cyberjack ]";
+      example = literalExpression "with pkgs; [ libykneomgr pcsc-cyberjack ]";
```

Comment:
The [home page][url] of libykneomgr says:

> This project is deprecated and is no longer being maintained.

[url]: https://developers.yubico.com/libykneomgr/

Mentioning a deprecated lib does not sound like a good idea.